### PR TITLE
feat: add pushgateway and rpc keepAlive options

### PR DESCRIPTION
**Pull request by JSHan94**

### CODE REVIEW

File: src/lib/monitor/l2.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export class L2Monitor extends Monitor {
 
     const pair = await config.l1lcd.ophost
       .tokenPairByL2Denom(this.bridgeId, data['denom'])
-      .catch((e) => {
+      .catch(() => {
```
Comment (Vritra4) [inline src/lib/monitor/l2.ts:60]: put a log when the exception catched

File: src/lib/metrics.ts [hunk 2]
```diff
@@ -34,6 +36,10 @@ interface AddMetricData {
   labels?: Partial<Record<string, string>>;
 }
 
+const pushgateway = new Pushgateway(config.PROMETHEUS_GATEWAY_URI, {
```
Comment (Vritra4) [inline src/lib/metrics.ts:39]: there's no need to construct a new Pushgateway when the mode is pull

File: src/lib/monitor/l2.ts [hunk 3]
```diff
@@ -57,7 +57,8 @@ export class L2Monitor extends Monitor {
 
     const pair = await config.l1lcd.ophost
       .tokenPairByL2Denom(this.bridgeId, data['denom'])
-      .catch(() => {
+      .catch((e) => {
+        this.logger.warn(`Failed to get token ${data['denom']} pair`)
```
Comment (Vritra4) [inline src/lib/monitor/l2.ts:61]: there's no e'd be used. please log e with lower log level

File: README.md [hunk 4]
```diff
@@ -84,6 +85,10 @@ TYPEORM_ENTITIES=dist/orm/*Entity.js      # entity path
 | EXECUTOR_L2_MONITOR_HEIGHT | L2 monitor start height (optional)         | 0                        |
 | ENABLE_API_ONLY            | Enable API only mode (optional)            | false                    |
 | SLACK_WEB_HOOK             | Slack web hook for notification (optional) | ''                       |
+| EXECUTOR_METRICS_PORT      | Executor metrics port                      | 8080                     |
+| PROMETHEUS_METRICS_MODE    | Prometheus metrics mode ['pull', 'push']   | 'pull'                   |
+| PROMETHEUS_GATEWAY_URI     | Prometheus push gateway URI                | 'https://127.0.0.1:9091' |
```
Comment (sh-cha) [inline README.md:90]: http

File: README.md [hunk 5]
```diff
@@ -96,6 +101,10 @@ TYPEORM_ENTITIES=dist/orm/*Entity.js      # entity path
 | BRIDGE_ID                 | Bridge ID                                  | ''                       |
 | OUTPUT_SUBMITTER_MNEMONIC | Mnemonic seed for output submitter         | ''                       |
 | SLACK_WEB_HOOK            | Slack web hook for notification (optional) | ''                       |
+| OUTPUT_METRICS_PORT       | Output metrics port                        | 8081                     |
+| PROMETHEUS_METRICS_MODE   | Prometheus metrics mode ['pull', 'push']   | 'pull'                   |
+| PROMETHEUS_GATEWAY_URI    | Prometheus push gateway URI                | 'https://127.0.0.1:9091' |
```
Comment (sh-cha) [inline README.md:106]: http

File: README.md [hunk 6]
```diff
@@ -117,6 +126,10 @@ TYPEORM_ENTITIES=dist/orm/*Entity.js      # entity path
 | CELESTIA_NAMESPACE_ID       | Celestia namespace id (optional)                             | ''                         |
 | PUBLISH_BATCH_TARGET        | Target chain to publish batch (supports: ['l1', 'celestia']) | 'l1'                       |
 | ENABLE_API_ONLY             | Enable API only mode (optional)                              | false                      |
+| BATCH_METRICS_PORT          | Batch metrics port                                           | 8082                       |
+| PROMETHEUS_METRICS_MODE     | Prometheus metrics mode ['pull', 'push']                     | 'pull'                     |
+| PROMETHEUS_GATEWAY_URI      | Prometheus push gateway URI                                  | 'https://127.0.0.1:9091'   |
```
Comment (sh-cha) [inline README.md:131]: http

**Review by sh-cha — APPROVED** [review]
small readme changes are required

**Pull request merged**